ingest/expression_files/dense_ingestor.py

```diff
@@ -28,6 +28,8 @@
 class DenseIngestor(GeneExpression, IngestFiles):
     # ToDo SCP-2635
     ALLOWED_FILE_TYPES = ["text/csv", "text/plain", "text/tab-separated-values"]
+    GENE_BATCH_SIZE = 5
+    DATA_ARRAY_BATCH_SIZE = 1000
```

Comment:
Devon, thanks for explaining the intent of `DATA_ARRAY_BATCH_SIZE`. Because this constant isn't intended to limit the number of data arrays that get loaded, it was helpful for me to understand how it was being used.